api/routes.php

```diff
@@ -16,6 +16,7 @@
     new Api\Route('get', '/template', Api\Template\ListTemplates::class),
     new Api\Route('get', '/template/{templateId}', Api\Template\GetTemplate::class),
     new Api\Route('get', '/template/{docType}/{templateKey}', Api\Template\GetTemplate::class),
+    new Api\Route('get', '/template/{docType}/{templateKey}/history', Api\Template\ListVersions::class),
```

Comment:
Consider changing the ListVersions class name to GetTemplateHistory.
- You aren't listing data. You're getting data.
- This revised name would make it consistent with the endpoint to get a template.